### ISSUE TITLE
A plugin that hides disabled emojis when you don't have nitro

### ISSUE THREAD

**Issue by Hedreon** [Vencord/plugin-requests#510]
### Plugin Description

Although [one issue already exists about this,](https://github.com/Vencord/plugin-requests/issues/440) the person who made the issue clearly stated that he wants to hide emotes that he doesn't want, rather than hiding all disabled emojis. ("Extremely close...but I want to pick what is hidden")

This plugin request is simple and I've been aching for something like this ever since I started using Vencord, being a former BetterDiscord user myself. It's basically a QoL plugin that makes picking your favourite emoji so much easier and basically saving you a *lot* of time (if you're in a considerable amount of Discord servers like me) by hiding emotes you currently don't have access to (because a) you don't have nitro and b) you're currently not in that server that has those emotes)

TL;DR: This is yap talk for a plugin that hides disabled emojis because you don't have nitro.

### Existing Plugin for other mod

Hide Disabled Emojis by Zerebos: https://betterdiscord.app/plugin/HideDisabledEmojis

### Request Agreement

- [X] I read the above and made sure my plugin is possible and allowed
- [X] I checked the [existing plugins](https://vencord.dev/plugins) and made sure my plugin doesn't already exist
- [X] I [searched the existing issues](https://github.com/Vencord/plugin-requests/issues?q=is%3Aissue) and made sure a similar request doesn't already exist

**Comment by Golonchy** [comment]
Try this
If it doesn't work copy the link put in ur browser and copy the code the plu it into a class updater then copy the new code and paste it in quickcss 
https://discord.com/channels/1015060230222131221/1028106818368589824/1050422723186270259

**Comment by Hedreon** [comment]
> Try this If it doesn't work copy the link put in ur browser and copy the code the plu it into a class updater then copy the new code and paste it in quickcss https://discord.com/channels/1015060230222131221/1028106818368589824/1050422723186270259

That doesn't hide disabled emojis

**Comment by Butterroach** [comment]
```css
li:has(button div[class^="emojiLockIconContainer"]), div[class*="categorySectionNitroLocked"],div[class^="nitroTopDivider"], div[class^="upsellContainer"] {
  display: none;
}
```

you WILL have a large emptiness before you get to the regular emojis (the ones that are like 🙂) and they WILL be a bit broken but still usable if you click the lil smiley on the left

**Comment by Hedreon** [comment]
> ```css
> li:has(button div[class^="emojiLockIconContainer"]), div[class*="categorySectionNitroLocked"],div[class^="nitroTopDivider"], div[class^="upsellContainer"] {
>   display: none;
> }
> ```
> 
> you WILL have a large emptiness before you get to the regular emojis (the ones that are like 🙂) and they WILL be a bit broken but still usable if you click the lil smiley on the left

That's exactly what I tried when trying to hide disabled emojis myself, even down to using the same classes you did. I'm serious when I say this cannot be fully done in just CSS.

**Comment by Butterroach** [comment]
> > ```css
> > li:has(button div[class^="emojiLockIconContainer"]), div[class*="categorySectionNitroLocked"],div[class^="nitroTopDivider"], div[class^="upsellContainer"] {
> >   display: none;
> > }
> > ```
> > 
> > you WILL have a large emptiness before you get to the regular emojis (the ones that are like 🙂) and they WILL be a bit broken but still usable if you click the lil smiley on the left
> 
> That's exactly what I tried when trying to hide the disabled emojis myself, even down to using the same classes you chose. I'm serious when I say this ***cannot*** be fully done in just CSS.

it works though

**Comment by Hedreon** [comment]
> > > ```css
> > > li:has(button div[class^="emojiLockIconContainer"]), div[class*="categorySectionNitroLocked"],div[class^="nitroTopDivider"], div[class^="upsellContainer"] {
> > >   display: none;
> > > }
> > > ```
> > > 
> > > you WILL have a large emptiness before you get to the regular emojis (the ones that are like 🙂) and they WILL be a bit broken but still usable if you click the lil smiley on the left
> > 
> > That's exactly what I tried when trying to hide the disabled emojis myself, even down to using the same classes you chose. I'm serious when I say this ***cannot*** be fully done in just CSS.
> 
> it works though

Works but doesn't solve the decluttering problem with the emoji picker hence the long time scrolling problem.
